<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string extracomment="mood"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addChumButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD CHUM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blockButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>BLOCK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pesterButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTreeView" name="chumsTree">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>58</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nameButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
      <horstretch>3</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>309</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="moodButton0">
       <property name="whatsThis">
        <string extracomment="mood"/>
       </property>
       <property name="text">
        <string>                      </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="moodButton20">
       <property name="whatsThis">
        <string extracomment="mood"/>
       </property>
       <property name="text">
        <string>                      </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="moodButton5">
       <property name="whatsThis">
        <string extracomment="mood"/>
       </property>
       <property name="text">
        <string>                      </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="moodButton21">
       <property name="whatsThis">
        <string extracomment="mood"/>
       </property>
       <property name="text">
        <string>                      </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="moodButton19">
       <property name="whatsThis">
        <string extracomment="mood"/>
       </property>
       <property name="text">
        <string>                      </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="moodButton22">
       <property name="whatsThis">
        <string extracomment="mood"/>
       </property>
       <property name="text">
        <string>                      </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="moodButton1">
       <property name="text">
        <string>                      </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="moodButton6">
       <property name="text">
        <string>                      </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="moodButton3">
       <property name="text">
        <string>                      </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
